### DISCUSSION TITLE
Examples in main readme

### DISCUSSION THREAD

**Discussion by baadev** [SandeepVashishtha/LaTeX-Resume-Builder#15] [Ideas]
I just checked out your project and thought it would be great if you included some resume screenshots in the main docs that will showcase the result of the resume builder. That way, users can quickly see what they can create and find exactly what they're looking for. 
I think it could help attract more users and maybe even contributors to the project

**Comment by SandeepVashishtha** [comment]
@baadev Thank you so much for checking out the project and for your valuable suggestion! I completely agree that adding resume screenshots in the main README would give users a clearer idea of what they can create with the resume builder. I'll work on updating the documentation with some example screenshots of the generated resumes soon. I appreciate your input, and hopefully, this will indeed attract more users and contributors.

Feel free to share any other feedback or ideas!
